From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-huge_memory-add-two-new-not-yet-used-functions-for-folio_split-fix-checkpatch-fixes
Date: Fri Feb 14 06:55:35 PM PST 2025

ERROR: trailing whitespace
#90: FILE: mm/huge_memory.c:3654:
+^I^I^I$

total: 1 errors, 0 warnings, 57 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

NOTE: Whitespace errors detected.
      You may wish to use scripts/cleanpatch or scripts/cleanfile

./patches/mm-huge_memory-add-two-new-not-yet-used-functions-for-folio_split-fix.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Zi Yan <ziy@nvidia.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/huge_memory.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/huge_memory.c~mm-huge_memory-add-two-new-not-yet-used-functions-for-folio_split-fix-checkpatch-fixes
+++ a/mm/huge_memory.c
@@ -3651,7 +3651,7 @@ after_split:
 
 			if (release == origin_folio)
 				continue;
-			
+
 			lru_add_page_tail(origin_folio, &release->page,
 						lruvec, list);
 
_
